girder_style/templates/account/base.html

```diff
@@ -3,68 +3,75 @@
 {% load account %}
 
 {% block body %}
-<body class="bg-gray-100 flex flex-col h-full relative">
+  <body class="bg-gray-100 flex flex-col relative">
+    <div class="flex flex-initial h-24 items-center justify-end relative w-full z-20">
+      {% if request.user.is_authenticated %}
+        <ul class="flex flex-grow h-full md:max-w-max lg:px-12">
+          <li class="flex flex-grow group relative">
+            <span class="cursor-pointer flex flex-row group items-center justify-between p-6 text-center text-primary-400 text-xs transition-all w-full hover:text-primary-300 med:justify-center">
+              <i class="ri-account-circle-fill opacity-75 mr-2 text-black text-xl transition-all group-hover:opacity-100"></i>
+              {{ request.user.email }}
+              <i class="ri-arrow-down-s-line opacity-50 ml-2 text-black text-xl transition-all"></i>
+            </span>
+            <ul class="absolute bg-white hidden left-2 overflow-hidden right-2 rounded shadow-md top-3/4 group-hover:block">
+              <li class="border-b border-gray-200 last:border-b-0">
```

Comment:
These were using `account/snippets/sub_menu_item.htm` before. Can they continue to use that? If not, that template snippet should be deleted.